explorer/interpreter/builtins.h

```diff
@@ -32,7 +32,15 @@ class Builtins {
     MulWith,
     ModWith,
 
-    Last = ModWith
+    // Bitwise and shift.
+    BitComplement,
+    BitAndWith,
+    BitOrWith,
+    BitXorWith,
+    LeftShiftWith,
+    RightShiftWith,
+
+    Last = RightShiftWith
```

Comment:
I will note, I'm wondering if it'd be better to just say "Last" (no assignment), requiring `< Last` instead of `<= Last`. That way this wouldn't need to be updated each time.

Reply:
The reason for this approach is to support covered `switch` checking -- with this approach, a `switch` over a builtin kind requires one `case` for each kind of buiiltin, and the compiler will complain if you get it wrong.